### DISCUSSION TITLE
MacOS Safari Extension

### DISCUSSION THREAD

**Discussion by TheRouletteBoi** [jordwest/news-feed-eradicator#400] [Ideas]
Can I request you port this over to a safari extension

**Comment by TheRouletteBoi** [comment]
After some research I was able to port the chrome extension to safari using this method [How to get Google Chrome Extensions in Safari](https://www.youtube.com/watch?v=V5UPRseAhq4) 


I've only tested on YouTube and it works great so far. 


![SCR-20240425-uze](https://github.com/jordwest/news-feed-eradicator/assets/9206290/ad4021de-eeb1-40bb-a59f-e9829b780ce4)

**Comment by TheRouletteBoi** [comment]
Bump, So you guys can sign the app and add it to the App Store

**Comment by cmeinerd** [comment]
+1 for Safari
